### PR TITLE
Show map from inside interior.

### PR DESCRIPTION
**Pull request by Pt-Djefferson**

### CODE REVIEW

File: The Long Dark Save Editor 2/Tabs/MapTab.xaml.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public partial class MapTab : UserControl
         private Point playerPosition;
 
         private string region;
+        private bool doNotRedrawMap;
```
Comment (FINDarkside) [inline The Long Dark Save Editor 2/Tabs/MapTab.xaml.cs]: I don't think this component actually needs to modify `Boot.m_SceneName` so if we remove it we don't need this.

**Comment by FINDarkside** [comment]
Maybe we should prevent the player from actually changing the location when indoors? I agree it's useful to see where you're on the map when you're indoors, but I'm not so sure if it's useful to being able to change your interior location according to the outside map. Even if some maps had accurate sizes of building, it wouldn't work too great as the exterior and interior building sizes don't often match. So a building can be bigger from the outside than it's from the outside :smile:

**Comment by Pt-Djefferson** [comment]
I didnt understand at all what you mean. This RP didnt allows to change indoor coords by map. It changes region aswell.

**Comment by FINDarkside** [comment]
Yeah my bad, didn't notice it changes the scene if you move the marker, that's good. Anyway, I think we should just get rid of `doNotRedrawMap` and rather just return early from the PropertyChanged handler if 
`region == MainWindow.Instance.CurrentSave.Boot.m_SceneName.Value`

**Comment by FINDarkside** [comment]
Nice work!

**Pull request merged**